Marlin/src/lcd/e3v2/proui/dwin.cpp

```diff
@@ -482,8 +471,7 @@ void dwinResetStatusLine() {
 // Djb2 hash algorithm
 uint32_t getHash(char * str) {
   uint32_t hash = 5381;
-  char c;
-  while ((c = *str++)) hash = ((hash << 5) + hash) + c; /* hash * 33 + c */
+  for (char c; (c = *str++);) hash = ((hash << 5) + hash) + c; /* hash * 33 + c */
```

Comment:
I just noticed that `getHash` doesn't seem to be used anywhere. I can't find any other references for it.

could then it be removed?? or whats it for?